src/sentry/static/sentry/app/components/card.tsx

```diff
@@ -0,0 +1,44 @@
+import {css} from '@emotion/core';
+import styled from '@emotion/styled';
+
+type Props = {
+  /**
+   * Adds hover and focus states to the card
+   */
+  withHover?: boolean;
```

Comment:
Maybe a better name is `interactive`?

Reply:
Maybe this is based on whether if a link is provided? hasLink

Reply:
What do you mean if a link is provided?